envoy-control-core/src/main/resources/lua/ingress_rbac_logging.lua

```diff
@@ -8,21 +8,26 @@ function envoy_on_request(handle)
     local request_id = first_header_value_from_list(request_id_header_names, handle)
     local trusted_header_name = handle:metadata():get("trusted_client_identity_header") or ""
     local client_name = ""
+    local trusted_client = "false"
     if trusted_header_name ~= "" then
         client_name = handle:headers():get(trusted_header_name) or ""
+        if client_name ~= "" and handle:connection():ssl() ~= nil then
+            trusted_client = "true"
+        end
     end
 
     if client_name == "" then
         client_name = first_header_value_from_list(client_identity_header_names, handle)
         if trusted_header_name ~= "" and client_name ~= "" and handle:connection():ssl() ~= nil then
-            client_name = client_name .. " (not trusted)"
+            client_name = client_name .. " (NOT TRUSTED)"
```

Comment:
why?

Reply:
Łukasz suggested, that it should be upper case like client missing

Reply:
ok, why not. But we should also change our internal docs. @slonka WDYT? Is this really needed in your opinion?

Reply:
i've abandoned this change, as it is not relevant if we will get rid of that suffix in near future